package.json

```diff
@@ -145,7 +145,7 @@
     "eventsource": "^2.0.2",
     "lodash": "^4.17.21",
     "randombytes": "^2.1.0",
-    "stellar-base": "10.0.0-soroban.4",
+    "stellar-base": "10.0.0-soroban.5",
```

Comment:
should this be changed as part of rel branch, assuming https://github.com/stellar/js-soroban-client/pull/117 merges first and does this and yarn.lock?

Reply:
yeah this'll get overwritten once #117 is in, thanks for noting it!